lib/core/network/repository/profile_repository.dart

```diff
@@ -22,12 +22,13 @@ class ProfileService {
         final map = Map<String, dynamic>.from(response.data);
         return Result.value(ProfileData.fromJson(map, user.network, []));
       } else {
-        LogHelper.e('get profile status error', stacktrace: StackTrace.current);
```

Comment:
this doesn't happen - a 404 ends in the catch block of this try/catch because dio throws